packages-content-model/roosterjs-content-model-core/lib/coreApi/paste.ts

```diff
@@ -30,6 +31,7 @@ export const paste: Paste = (
     pasteType: PasteType = 'normal'
 ) => {
     core.api.focus(core);
+    addUndoSnapshot(core, false);
```

Comment:
we should have already taken undo snapshot when call `formatContentModel` from `mergePasteContent`, so I don't think we need to add undo snapshot here.